draft-issuance.md

```diff
@@ -0,0 +1,162 @@
+
+## Issuance Calculation
+
+Given the block height `h` define a function **BlockSubsidy(h)**, such that:
+
+**BlockSubsidy(h)** = Block subsidy for a given `h`, that satisfies above requirements.
+
+Using an exponential decay function for **BlockSubsidy** satisfies requirements **R1** and **R2** above:
+
+`BlockSubsidy(h) = BLOCK_SUBSIDY_FRACTION * ZsfBalanceAfter(h - 1)`
+
+Finally, to satisfy **R3** above we always round up to the next zatoshi.
+
+`BlockSubsidy(h) = ceiling(BLOCK_SUBSIDY_FRACTION * ZsfBalanceAfter(h - 1))`
+
+# Rationale
+
+## `BLOCK_SUBSIDY_FRACTION`
+
+Let `IntendedZSFFractionRemainingAfterFourYears` = 0.5.
+
+The value `41 / 100_000_000` satisfies the approximation:
+
+`(1 - BLOCK_SUBSIDY_FRACTION)^PostBlossomHalvingInterval ≈ IntendedZSFFractionRemainingAfterFourYears`
+
+Meaning after a period of 4 years around half of `ZSF_BALANCE` will be paid out
+as block subsidies, thus satisfying **R4**.
+
+## Visualization of the Smoothed Curve
+
+The following graph, taken from the ECC blog post, illustrates the smoothed curve. Note that depending on when the network upgrade takes place the disbursement may temporarily _increase_.
+
+![A graph showing a comparison of the halving-based step function vs the smoothed curve](./draft-zip-smoothed-issuance-curve.png)
+
+## Other Notes
+
+The suggested implementation avoids using float numbers. Rust and C++ will both round
+the result of the final division up, satisfying **R3** above.
+
+# Appendix: Simulation
+
+We encourage readers to run the following Rust code, which simulates block subsidies.
+According to this simulation, assuming no deflationary action, block subsidies would
+last for approximately 113 years:
+
+## Rust Code
+
+```rust
+fn main() {
+    // approximate available subsidies in August of 2023
+    let mut available_subsidies: i64 = 4671731 * 100_000_000;
+    let mut block: u32 = 0;
+
+    while available_subsidies > 0 { 
+        let block_subsidy = (available_subsidies * 41 + 99_999_999) / 100_000_000;
+        available_subsidies -= block_subsidy;
+
+        println!(
+            "{} ({} years): {}({} ZEC) {}({} ZEC)",
+            block,                             // current block
+            block / 420_768,                   // ~ current year
+            block_subsidy,                     // block subsidy in zatoshis
+            block_subsidy / 100_000_000,       // block subsidy in ZEC
+            available_subsidies,               // available subsidies in zatoshis
+            available_subsidies / 100_000_000  // available subsidies in ZEC
+        );
+
+        block += 1;
+    }   
+}
+```
+
+Last line of output of the above program is:
+
+`47699804 (113 years): 1(0 ZEC) 0(0 ZEC)`
+
+Note the addition of 99,999,999 before division to force rounding up of non-zero values.
+
+
+# References
+
+[1] RFC-2119: https://datatracker.ietf.org/doc/html/rfc2119
+
+[2] ZIP-200: https://zips.z.cash/zip-0200
+
+[3] ZIP-XXX: Placeholder for the ZSF ZIP
```

Comment:
This also needs `[#protocol-diffadjustment]`.

Reply:
@daira is that a note for the ZIP owners? If so, could you clarify? Do I just need to add the anchor to the URL?